tests/scripts/deploy_admission_controller.sh

```diff
@@ -3,24 +3,59 @@
 # Sets up the environment for the admission controller webhook in the active cluster.
 set -eo pipefail
 
+trap cleanup EXIT SIGINT ERR
+function cleanup() {
+  kubectl -n rook-ceph delete validatingwebhookconfigurations $WEBHOOK_CONFIG_NAME
+  kubectl -n rook-ceph delete certificate serving-cert
+  kubectl -n rook-ceph delete issuers selfSigned-issuer
+  kubectl delete -f https://github.com/jetstack/cert-manager/releases/download/$CERT_VERSION/cert-manager.yaml
+}
+
 # Set our known directories and parameters.
 BASE_DIR=$(cd "$(dirname "$0")"; pwd)
+CERT_VERSION="v1.2.0"
 
 # Set the variables for webhook. Any change here for namespace and service name should be done in pkg/operator/admission/init.go also
 [ -z "${NAMESPACE}" ] && NAMESPACE="rook-ceph"
 WEBHOOK_CONFIG_NAME="rook-ceph-webhook"
 SERVICE_NAME="rook-ceph-admission-controller"
 
-INSTALL_SELF_SIGNED_CERT=true
 echo "$BASE_DIR"
 
-if [ "${INSTALL_SELF_SIGNED_CERT}" == true ]; then
-	"${BASE_DIR}"/webhook-create-signed-cert.sh --namespace ${NAMESPACE}
-fi
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/$CERT_VERSION/cert-manager.yaml
+timeout 150 sh -c 'until [ $(kubectl -n cert-manager get pods --field-selector=status.phase=Running|grep -c ^cert-) -eq 3 ]; do sleep 1; done'
+timeout 10 sh -c 'until [ $(kubectl -n cert-manager get pods -o custom-columns=READY:status.containerStatuses[*].ready | grep -c true) -eq 3 ]; do sleep 1; done'
+
 echo "Deploying webhook config"
 export NAMESPACE
 export WEBHOOK_CONFIG_NAME
 export SERVICE_NAME
+
+cat <<EOF | kubectl create -f -
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: selfsigned-issuer
+  namespace: ${NAMESPACE}
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: serving-cert
```

Comment:
How about a more descriptive name like this?
```suggestion
  name: rook-admission-controller-cert
```

Reply:
done.